hirte.spec.in

```diff
@@ -91,6 +92,38 @@ This package contains the node agent.
 %{_unitdir}/hirte-dep@.service
 %{_userunitdir}/hirte-dep@.service
 
+%package selinux
+Summary:  Hirte SELinux policy
+BuildRequires: checkpolicy
+BuildRequires: selinux-policy-devel
+
+%if "%{_selinux_policy_version}" != ""
+Requires: selinux-policy >= %{_selinux_policy_version}
```

Comment:
Should we `Requires: selinux-policy` if not specific version is specified? (ie: in a else clause?)

Reply:
AFAIK you need to require the exact selinux-policy with which you built it. In theory even the newer one should work, but we already had a case in oVirt where this didn't work.

Anyway the above is the official recommendation:
https://fedoraproject.org/wiki/SELinux_Policy_Modules_Packaging_Draft#Runtime_Dependencies

Reply:
ah ok, so the macro is prepopulated by `selinux-policy` cool, thanks for the info!